cmd/terramate/cli/project.go

```diff
@@ -225,24 +226,37 @@ func (p project) checkDefaultRemote() error {
 	)
 }
 
-func (p *project) checkLocalDefaultIsUpdated() error {
+func (p *project) checkRemoteDefaultBranchIsReachable() error {
 	gitcfg := p.gitcfg()
+
+	remoteDesc := fmt.Sprintf("remote(%s/%s)", gitcfg.DefaultRemote, gitcfg.DefaultBranch)
+
+	logger := log.With().
+		Str("HEAD", p.headCommit()).
+		Str(remoteDesc, p.remoteDefaultCommit()).
```

Comment:
context fields should not have special chars.

```suggestion
		Str("default_branch", remoteDesc).
		Str("default_hash", p.remoteDefaultCommit()).
```